functions/index.js

```diff
@@ -0,0 +1,23 @@
+const logger = require('./middleware/logger');
+const express = require('express');
+const config = require('config');
+
+
+const functions = require('firebase-functions');
+const admin = require('firebase-admin');
+// Initialize Firebase App
+admin.initializeApp({
+  credential: admin.credential.applicationDefault(),
+  databaseURL: "https://local-parikrama.firebaseio.com"
+});
+
+const categories = require('./api/category');
+const products = require('./api/product');
+logger.info('Initializing Inventory Management System');
+// Initialize express
+const app = express();
+require('./startup/routes')(app);
+
+exports.app = functions.https.onRequest(app)
```

Comment:
i have a suggestion here. Can we name this export as "api" and get rid of /api in the startup . So that final url would look : http://localhost:5001/exercise1-ea6c9/us-central1/api/users

Reply:
The final url after deployment is this: https://local-parikrama.web.app/api/products. I think it doesn't matter after deployment.  It will be /api/users/